lib/fluent/logger/fluent_logger.rb

```diff
@@ -49,6 +49,10 @@ def self.from_msgpack_ext(data)
       def to_json(*args)
         @sec.to_s
       end
+
+      def as_json(*args)
+        @sec
+      end
```

Comment:
It seems we should leave a comment on why we need this.

Reply:
Added a comment.
https://github.com/fluent/fluent-logger-ruby/pull/112/commits/ec11c6db74b9867b042ee63a39d2d1ef9f589e6e